tests/exchanges/util/test_exchange_market_status_fixer.py

```diff
@@ -164,7 +164,7 @@ def test_exchange_market_status_fixer_with_str_instead_of_floats(self):
         ).market_status == {
             Ecmsc.PRECISION.value: self._get_precision(None, None, None),
             # replace "plop" and nan with computed numbers
-            Ecmsc.LIMITS.value: self._get_limits(0.05, 1e4, 65.99999999999999, 1111111.1111, 3.3, 11111111111)
+            Ecmsc.LIMITS.value: self._get_limits(0.05, 1e4, None, None, 3.3, 11111111111)
```

Comment:
Isn't it an issue to have None values here?

Reply:
It's handled in `decimal_check_and_adapt_order_details_if_necessary`, I added tests as well. When price is None, it means there is no minimum / maximum price

Reply:
Ok Nice :+1: